util/pool.c

```diff
@@ -245,9 +247,12 @@ pool_t *pool_setalloc_verify_init(int entry_size, int stepsize,
 #endif
     if (verify)
         entry_size += sizeof(pool_t *);
-    entry_size = (entry_size + 3) & (-1 ^ 3);
-    if (entry_size < 4)
-        entry_size = 4;
+    /* must store a next-ptr in the block when it's on the free list */
+    entry_size = ALIGN(entry_size, sizeof(struct freeblk));
```

Comment:
Would the word size be good enough here (eg sizeof(void *))?

Reply:
It would be enough in this case, since the struct currently only has a single pointer. Using the struct size instead avoids breaking things if the struct changes shape in the future for some reason.